oci-library/oci-hol/java-management/set-up-of-management-agent/set-up-of-management-agent.md

```diff
@@ -268,28 +306,87 @@ Install Management Agent (If your host is Windows, skip to **For Windows** Secti
 
-7. In the Oracle Cloud Console, open the navigation menu and click **Observability & Management**. Click **Java Management**.
+7. In the Oracle Cloud Console, open the navigation menu, click **Observability & Management**, and then click **Fleets** under **Java Management**.
 
-  ![image of console navigation to java management](/../images/console-navigation-java-management.png)
+  ![image of console navigation to java management](/../images/console-navigation-jms.png)
 
 8. Select the compartment that the fleet is in and click the fleet.
 
 9. Click on **Modify Agent Settings**.
 
   ![image of fleet details page](/../images/fleet-details-page-new.png)
 
-10. Change the **Java Runtime Discovery** and **Java Runtime Usage** to the desired value.
+10. Change the **Java Runtime Discovery** and **Java Runtime Usage** to the desired value. For this example, change **Java Runtime Discovery** to **3 hours**, and **Java Runtime Usage** to **5 minutes**.
 
   ![image of modify agent settings page](/../images/fleet-modify-agent-settings-new.png)
 
-11. If tagging and installation of management agents is successful, Java Runtimes will be indicated on the Fleet Main Page.
+## Task 6: Verify detection of Java applications and runtimes
+For the logging of applications to be visible, Java applications must be run again after the installation of the Management Agent. Now that the Management Agent has been set up in your compute instance, it will be able to detect new Java applications that have been executed. This can be observed in the Oracle Cloud Console.
+
+We shall demonstrate the detection of the Java compiler and HelloWorld application created in Lab 2.
+1. First, re-compile the HelloWorld.java file:
+
+    ```
+    <copy>
+    javac HelloWorld.java
+    </copy>
+    ```
+
+    Then execute the HelloWorld application:
+
+    ```
+    <copy>
+    java HelloWorld
+    </copy>
+    ```
+
+2. In the Oracle Cloud Console, open the navigation menu, click **Observability & Management**, and then click **Fleets** under **Java Management**.
+
+  ![image of console navigation to java management](/../images/console-navigation-jms.png)
+
+3. Select the compartment that the fleet is in and click the fleet.
+
+4. Click **Java Runtimes** under **Resources**. If tagging and installation of management agents is successful, Java Runtimes will be indicated on the Fleet Main Page after 5 minutes.
+
+  You should see only one Java Runtime. This corresponds to the Java 8 installation from Lab 2.
 
   ![image of successful installation](/../images/successful-installation.png)
 
+12. Click **Applications** under **Resources**. You should now see two applications. The first is from the javac compiler command and the second is from the HelloWorld application.
+
+  ![image of applications after successful installation](/../images/successful-installation-applications.png)
+
+  You may now **proceed to the next lab.**
+
 ## Troubleshoot Management Agent Installation Issues
 
```

Comment:
For troubleshooting mgmt agent issues, there is mention of Task 1, 2, 3 helpful tips that will help them to troubleshoot. It may be good to mention at Task 1 2 3 respectively that if they face any issues they can look downwards because if there is no mention they might get stuck there and stop here.

Reply:
good point! will add it in accordingly